ui/GeoCounter.cs

```diff
@@ -0,0 +1,116 @@
+using Godot;
+using System;
+
+public partial class GeoCounter : Control
+{
+	[Export]
+	public AnimatedSprite2D GeoIcon;
+	[Export]
+	public Label PlusSignLabel;
+	[Export]
+	public Label CurrentGeoLabel;
+	[Export]
+	public Label GeoToAddLabel;
+
+	private bool IsGeoTransferring = false;
```

Comment:
For private variables it's usually common practice to add an '_' (underscore) before the variable name. This is a common quick indicator that it is a private variable. So, for example this would be `_isGeoTransferring`